blockly-core/core/ui/block_space/flyout.js

```diff
@@ -358,13 +358,6 @@ Blockly.Flyout.prototype.hide = function() {
     Blockly.unbindEvent_(this.reflowWrapper_);
     this.reflowWrapper_ = null;
   }
-  // Delete all the blocks.
-  var blocks = this.blockSpace_.getTopBlocks(false);
-  for (var x = 0, block; block = blocks[x]; x++) {
-    if (block.blockSpace == this.blockSpace_) {
```

Comment:
In the extracted version I removed this filter since it's assumed elsewhere this will always be the case for the flyout's `blockSpace_.getTopBlocks`. Some manual testing seemed to confirm this